Doc/tutorial/introduction.rst

```diff
@@ -315,9 +315,9 @@ the second row gives the corresponding negative indices. The slice from *i* to
 *j* consists of all characters between the edges labeled *i* and *j*,
 respectively.
 
-For non-negative indices, the length of a slice is the difference of the
-indices, if both are within bounds.  For example, the length of ``word[1:3]`` is
-2.
+The length of a slice is the difference of the
```

Comment:
It is more complex than that. Please, take a look at this example:

```python
>>> len(range(2, -2, -1))
4
```

Source: https://github.com/python/cpython/blob/f878d46e5614f08a9302fcb6fc611ef49e9acf2f/Objects/rangeobject.c#L224

Reply:
But it is about slices of strings, not range object. We can talk about that in range reference

Reply:
I.e: length of range object equals math.ceil((stop - start) / step)

Reply:
oh, sorry, it was too late for me to do a review 😆